clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -186,7 +186,11 @@ public DaVinciBackend(
           new VeniceWriterFactory(backendProps.toProperties(), pubSubClientsFactory.getProducerAdapterFactory(), null);
       String instanceName = Utils.getHostName() + "_" + Utils.getPid();
       int derivedSchemaID = backendProps.getInt(PUSH_STATUS_STORE_DERIVED_SCHEMA_ID, 1);
-      pushStatusStoreWriter = new PushStatusStoreWriter(writerFactory, instanceName, derivedSchemaID);
+      Map<String, VeniceWriterFactory> writerFactoryMap = new HashMap<>(1);
+      String clusterName = backendConfig.getClusterName();
```

Comment:
DVC is cluster agnostic and can discover cluster using its NativeMetadataRepository since a DVC can be subscribed to multiple stores belonging to different Venice clusters. You can use the `ClusterInfoProvider` interface to get cluster name given a store name.

Reply:
For DVC and server side, we do not need clusterName discovery logic. As each host naturally belong to a specific cluster.

Reply:
That assumption is true for server but how is it true for DVC? What if the DVC user subscribe to multiple stores that span across multiple venice clusters?

Reply:
That is a valid concern.

Reply:
This comment is kind of old, but I think it's pretty valid.  Are we going to address this?

Reply:
Since this cluster by cluster is mainly used for controller only. I do not plan to do it in this PR, but the concern is valid, we can do it in future.

Reply:
Hmm, sorry I am not sure if I am fully follow your intention of this PR, if this is only for controller, I think you could skip DVC for now? As for seas-careers, their DVC use cases are spanning across different clusters already. This might not be correct for them. 
(Some of the config in backend config does not apply to all DVC stores)

Reply:
Yeah I don't see a good reason here to introduce a potential bug into a client library that even if harmless we'll have to force everyone to upgrade off of at some point.